config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-gcs-to-aws-sync.yaml

```diff
@@ -29,7 +29,7 @@ periodics:
             - |
               aws sts get-caller-identity
               echo "initiating a sync between GCS and S3..."
-              ./hack/sync-to-s3.sh
+              ./hack/tools/sync-to-s3.sh
```

Comment:
maybe we can override the source bucket with an env so we can do this without waiting?

Reply:
hmm:
- I need someone to initialise the registry
- Then update this script(the proposed PR in k/registry.k8s.io updates it)
- Move it to registry.k8s.io repo(the registry.k8s.io folder in k/k8s.io repo needs to be emptied out before the end of the Month so I can move the contents of k8s.gcr.io in k/k8s.io repo to it)

Reply:
I mean moving the script between repos could be merged first so iterative changes to the script are clearer to review (since they'll be diffs on top of the move diff), if we moved the script today without waiting for the rest.

1. move script
2. spin up new locations
3. patch script / override settings for new locations